packages/sdk/src/client.ts

```diff
@@ -2193,6 +2193,7 @@ export class Client
             miniblocks,
             'backward',
         )
+        await this.persistenceStore.saveSnapshots(streamIdAsString(streamId), snapshots)
 
```

Comment:
We don't want to save these snapshots in persistence - the client only cares about the latest snapshot.

Reply:
we do save the snapshots rolling forward though... 
[https://github.com/towns-protocol/towns/blob/638fc7a132adb1b8cfd34f3b990e7ddb93c6e4a3/packages/sdk/src/syncedStream.ts#L1](https://github.com/towns-protocol/towns/blob/638fc7a132adb1b8cfd34f3b990e7ddb93c6e4a3/packages/sdk/src/syncedStream.ts#L172)